www/help-review-votes

```diff
@@ -49,6 +50,11 @@ responsibly.  <b>Don't</b> flag a review for spoilers that are already
 properly hidden with "click to see" warnings - only flag it if the
 spoilers are out in the open.
 
+<p><b>Flag as inappropriate:</b> Notify the moderators that this 
+review may violate the 
+<a href="https://ifdb.org/code-of-conduct">Code of Conduct</a> or the 
+<a href="https://ifdb.org/tos">Terms of Service</a>.
```

Comment:
Generally we should use relative links, like `<a href="/code-of-conduct">` instead of absolute links to ifdb.org, so the links will DTRT when testing on local machines or on the dev site.

(I'll fix it.)